xc/xc7/archs/zynq7_z020/tiles/clbll_l/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+project_ray_tile(
+  ARCH zynq7_z020
+  TILE CLBLL_L
+  SITE_TYPES SLICEL/SLICEL0 SLICEL/SLICEL1
+  )
```

Comment:
I believe there was an error in the original PR I made for the zynq boards addition. There was a change in the tiles used here: https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1484. This basically avoids using SLICEs as tiles, and reintroduces CLBs.

All the CLB tiles would require to be in the same format as the other archs:
https://github.com/SymbiFlow/symbiflow-arch-defs/blob/5588c2891f4d773cbbeb85843626fd4972bebeee/xc/xc7/archs/zynq7/tiles/clbll_l/CMakeLists.txt#L1-L5